### ISSUE TITLE
[TreeView] isExpanded doesn't function as expected

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#3245]
**Describe the bug**
When using the kendoTreeViewHierarchyBinding and isExpanded is set to return always true, then the items aren't expanded.

It looks like the issue is caused by the [isVisible] property introduced in v 5.1.0.

Please find some additional details in thread 1511268.

**To Reproduce**
Please see the following local project (the issue isn't reproduced in StackBlitz)
[treeview.zip](https://github.com/telerik/kendo-angular/files/6195430/treeview.zip)

**Comment by geo242** [comment]
I am seeing this in the flat binding as well. It appears that the `isVisible` input is required, otherwise all nodes will be hidden. It's easy enough to add a simple function that always returns `true` if you aren't using filtering. However, I would prefer that it be an optional input and the default not be to hide the nodes. This should not have been a breaking change.

**Comment by gkarapeev** [comment]
We've identified the issue. It has to do with the order in which the inputs are given to the TreeView component.

**Not working**
```
[nodes]="data"
[childrenField]="'items'"
```

**Working**
```
[childrenField]="'items'"
[nodes]="data"
```

This is definitely an issue we will fix as soon as possible, but for the time being, a workaround would be to just reorder the inputs of the component.

**Comment by gkarapeev** [comment]
@geo242 thank you for joining the discussion. Could you please provide a snippet of the TreeView input bindings you have, where this issue is reproducible? I was only able to reproduce this with the hierarchy binding directive.

**Comment by ipeshev** [comment]
Fix available in [latest dev version](https://www.telerik.com/kendo-angular-ui-develop/components/installation/development-builds).

**Comment by dimitar-pechev** [comment]
Shipped with `@progress/kendo-angular-treeview@5.4.0`.
